pkg/operator/ceph/webhook-config.go

```diff
@@ -0,0 +1,233 @@
+/*
+Copyright 2022 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package operator
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	api "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
+	v1 "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
+	"github.com/rook/rook/pkg/clusterd"
+
+	admv1 "k8s.io/api/admissionregistration/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	issuerName        = "selfsigned-issuer"
+	certificateName   = "rook-admission-controller-cert"
+	webhookConfigName = "rook-ceph-webhook"
+)
+
+func createIssuer(ctx context.Context, clusterdContext *clusterd.Context) (*api.Issuer, error) {
```

Comment:
@travisn continuing discussion, I was talking about this issuer and certs(creating below)

Reply:
Is the question whether the operator should remove this cert config during uninstall? Could we set the owner reference to the operator deployment? If the operator goes away, then the resources would go away. When the operator is created again, then the operator would have to create them again.